slam-libraries/viam-orb-slam3/orbslam_server_v1.cc

```diff
@@ -110,6 +110,86 @@ ::grpc::Status SLAMServiceImpl::GetPositionNew(
     return grpc::Status::OK;
 }
 
+::grpc::Status SLAMServiceImpl::GetPointCloudMap(
```

Comment:
This is equal to this if statement https://github.com/viamrobotics/slam/pull/139/files#diff-b68d96da1bc970406f0f4402811d582812550e7c8450a636d73540f0363c73d8L318, just fitting it into a `GetPointCloudMapResponse` return value.

Reply:
+ a refactor to remove the color information (as it is not longer meaningful) & refactor to only use a single string type to create the pointcloud pcd, with helper functions where possible.